contrib/pg_tde/meson.build

```diff
@@ -115,6 +115,8 @@ tap_tests = [
   't/011_unlogged_tables.pl',
   't/012_replication.pl',
   't/013_crash_recovery.pl',
+  't/014_pg_waldump_basic.pl',
+  't/015_pg_waldump_fullpage.pl',
```

Comment:
shouldn't we also add them to the make build?

Reply:
You do not have to. The make build scans the directory for files so TAP tests are enabled with just `TAP_TESTS = 1`.

Reply:
Of course, I completely forgot that just looking at the diff